app/assets/stylesheets/examples.scss

```diff
@@ -0,0 +1,179 @@
+  text-align: center;
+}
+
+.row-examplepage-middle {
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 30px;
+  text-align: center;
+}
+
+.card-examplepage-example {
+  border: 1px solid $green;
+  border-radius: 0;
+  display: inline-block;
+  margin-bottom: 30px;
+  margin-top: 30px;
+  width: 98%;
+}
+
+.card-img-examplepage-example {
+  max-height: 250px;
+}
+
+.card-footer-examplepage-example {
+  background-color: $white;
+}
+
+.examplepage-example-p {
+  display: inline-block;
+  float: left;
+  font-weight: 600;
+  margin-top: 10px;
+  overflow: hidden;
+  text-align: left;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 80%;
+}
+
+.examplepage-example-button {
+  float: right;
+  margin-bottom: 15px;
+
+  &:hover {
+    background-color: $teal;
+    box-shadow: 0 8px 15px $shadow-black;
+    color: $white;
+    transform: translateY(-7px);
+  }
+}
+
+.btn-examples {
+  background-color: $green;
+  border-radius: 30px;
+  color: $white;
+  margin: 30px 10px;
+  text-align: justify;
+  transition: all .3s ease 0s;
+
+  &:hover {
+    background-color: $teal;
+    box-shadow: 0 8px 15px $shadow-black;
+    color: $white;
+    transform: translateY(-7px);
+  }
+}
+
+.btn-examples-light {
+  background-color: $white;
+  border-radius: 30px;
+  color: $white;
+  margin: 30px 10px;
+  text-align: justify;
+  transition: all .3s ease 0s;
+
+  &:hover {
+    background-color: $teal;
+    box-shadow: 0 8px 15px $shadow-black;
+    color: $white;
+    transform: translateY(-7px);
+  }
+}
+
+.btn-examples-close {
+  background-color: $green;
+  color: $white;
+  float: right;
+  font-size: 12px;
+  margin-bottom: 12px;
+  margin-left: 2px;
+  margin-top: 12px;
+  padding-bottom: 4px;
+  padding-left: 6px;
+  padding-right: 6px;
+  padding-top: 4px;
+  text-align: justify;
+  transition: all .3s ease 0s;
+}
+
+.modal-title {
+  color: $green;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.container-examples {
+  background-color: $gray-100;
+  font-family: Open Sans;
+  padding-top: 50px;
+}
+
+//styling for search bar
+
+.nav-search-form {
+  background-color: $white;
+  border: 1px solid $teal;
+  border-radius: 30px;
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+}
+
+.nav-bar-search-input {
+  border: 0;
+  border-radius: 30px;
+  outline: 0;
+  padding-left: .8rem;
+}
+
+.ns {
+  margin-bottom: 40px;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+}
+
+.search-logo {
+  background-color: $white;
+  border-radius: 30px;
+  height: 33.2px;
+  left: 0;
+  position: absolute;
+  top: 0;
+  width: 39.6px;
+  z-index: 9;
+}
+
+.search-type {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background-color: $white;
+  border-radius: 30px;
+  cursor: pointer;
+  font-size: .8rem;
+  height: 33.2;
+  padding-right: 1.6rem;
+  width: 39.6;
+}
+
+.fa {
+  left: 8px;
+  position: relative;
+  text-align: center;
+  top: 4px;
+}
```

Comment:
- Do not override core classes from bootstrap and font-awesome here
- Avoid having generic name for classes for a particular page
- Ensure that custom styles added are not replaceable with builtin bootstrap classes